src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -150,6 +150,45 @@ async function getAdditionalFacilityInfo(futureAppointments, useV2 = false) {
   return facilityData;
 }
 
+/**
+ * Wrapper function to retrieve facility information from the appointment
+ * record when using the v2 api or make api call to retrieve facility
+ * information.
+ *
+ * @param {*} appointments
+ * @param {*} dispatch
+ * @param {*} featureFacilitiesServiceV2
+ */
+async function getAdditionalFacilityInfoV2(
+  appointments,
+  dispatch,
+  featureFacilitiesServiceV2,
```

Comment:
I missed this before, I don't think this is the feature flag we want to check. I think we want to check `vaOnlineSchedulingVAOSServiceVAAppointments` for appointments and `vaOnlineSchedulingVAOSServiceRequests` for requests. Those two flags indicate that we're fetching from vaos/v2/appointments, which is the endpoint that has the facility data.

You can see this by turning off the VAOS service feature flags, but leaving `vaOnlineSchedulingFacilitiesServiceV2`, you'll see the appointment list and the VA appointments will say "VA appointment" instead of the facility name.